### DISCUSSION TITLE
How to verify a JWT ignoring the expiry

### DISCUSSION THREAD

**Discussion by varuna-aample** [panva/jose#258] [Q&A] [answered]
I'm looking for a way to verify a JWT ignoring its expiration claim, I couldn't find any such options for the `jwtVerify()`
is there a way to verify ignoring the expiry of the token?

**Comment by panva** [comment, marked as answer]
You can either validate the JWT in full using jwtVerify, or just it's signature using compactVerify. There's no middle ground available.

**Reply by varuna-aample** [reply]
Thank you for the clarification.
